src/modules/rating/controllers/admin-rating.controller.ts

```diff
@@ -68,25 +68,29 @@ export class AdminRatingController {
 		@Query('status') status: string,
 		@Query('cursor') cursor?: string | null,
 		@Query('size') size: number = 10,
+		@Query('sortOrder') sortOrder: 'asc' | 'desc' = 'desc', // Default to descending order for newest to oldest
 	): Promise<FilteredByStatusResponseDTO> {
 		try {
 			let ratings: Rating[]
-
 			if (cursor === '') {
-				ratings = await this.ratingRepo.getByStatus(status, null, size)
+				ratings = await this.ratingRepo.getByStatus(
+					status,
+					null,
+					size,
+					sortOrder,
+				)
 			} else {
 				ratings = await this.ratingRepo.getByStatus(
 					status,
 					cursor,
 					size,
+					sortOrder,
 				)
 			}
-
 			let newCursor: string | null = null
-
 			if (ratings.length > 0) {
 				if (ratings.length > size) {
-					newCursor = ratings[size]._id
+					newCursor = ratings[size].createdAt.toISOString() // Convert createdAt Date to ISO string
```

Comment:
What is the purpose of this update ?

Reply:
this update is chosing the lastest created date of the rating for more details created time using ISOstring

Reply:
That's not the purpose of **newCursor**. **newCursor** is used to store the starting point of the query. For example:
- When user opens the rating page, it loads first 10 ratings. The starting point is _null_.
- When user clicks on **Next** button, the page should load next 10 ratings. The starting point should be the ID of 11th rating.